assets/lang/pl_PL.ini

```diff
@@ -81,8 +81,8 @@ OnScreen = Przyciski ekranowe
 Portrait = Pionowo
 Portrait Reversed = Odwrócone pionowo
 PSP Action Buttons = Klawisze akcji PSP
-Raw input = Raw input
-Repeat mode = Repeat mode
+Raw input = Zwykłe wejście
```

Comment:
Nie wiem czy jest sens tłumaczenia Raw, przetłumaczone na "zwykłe" nie dość, że niepoprawne, to jeszcze nic nikomu nie powie.

Reply:
Problem w tym, że (iirc) było to w dość widoczyn miejscu w ustawieniach i pozostawienie tam `RAW Input` również do mnie nie przemawia...
Może `Standardowe` lub `bezpośrednie`?